src/common/gui/RuntimeFont.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "vstgui/vstgui.h"
+#include "vstgui/lib/cfont.h"
+#include "vstgui/lib/platform/iplatformfont.h"
+
+namespace Surge
+{
+namespace GUI
+{
+
+/**
+ * initializeRuntimeFontFromDistribution
+ *
+ * The role of this function is to load a font from the bundle, dll, so whatever
+ * and plumb through the VSTGUI to create the 9 and 14 point fonts onto the two
```

Comment:
I don't understand what kind of act it is when you "plumb through".  I would start a fresh new sentence instead of a clause "The loaded font is ..." and explain preferably roughly in one or two sentences what is done with VSTGUI.